src/Proxies/GoogleGtagJs.php

```diff
@@ -13,16 +13,20 @@ class GoogleGtagJs {
 	/** @var array */
 	private $wcga_settings;
 
+	/** @var bool $ga4c_v2 True if Google Analytics for WooCommerce version 2 or higher is installed */
+	private $ga4w_v2;
+
 	/**
 	 * GoogleGtagJs constructor.
 	 *
 	 * Load the WooCommerce Google Analytics for WooCommerce extension settings.
 	 */
 	public function __construct() {
 		$this->wcga_settings = get_option( 'woocommerce_google_analytics_settings', [] );
+		$this->ga4w_v2       = defined( '\WC_GOOGLE_ANALYTICS_INTEGRATION_VERSION' ) && version_compare( \WC_GOOGLE_ANALYTICS_INTEGRATION_VERSION, '2.0.0', '>=' );
```

Comment:
:nail_care: 
We could reuse this property in https://github.com/woocommerce/google-listings-and-ads/blob/289230e818ed4b4d68888662102c2d706eecd17e/src/Google/GlobalSiteTag.php#L220:L220